solutions/standard/catalogValidationValues.json.template

```diff
@@ -2,5 +2,6 @@
   "ibmcloud_api_key": $VALIDATION_APIKEY,
   "region": "us-south",
   "resource_tags": $TAGS,
-  "resource_group_name": $PREFIX
+  "resource_group_name": $PREFIX,
+  "enable_cos_kms_encryption": false
```

Comment:
I think we need to explore this part some more because AFAIK this file will only hold the mandatory variables.
IMHO, I think `region` also should not be a part of this file as we do have a default value for `region` but I understand we need to be consistent too.

In this case,  `enable_cos_kms_encryption` too should not be a part of `catalogValidationValues.json.template` as it is not a mandatory value.

Please let me know your thoughts as well.

Reply:
In this DA code, the default value of `enable_cos_kms_encryption` is true so KP instance crn is mandatory to provide.
To address this, 2 approaches are there:
- either provide KP instance crn
- or set enable_cos_kms_encryption as false. `true` value is being tested in pr_tests.
Hence I did this after discussing with Conall.